### PR TITLE
Milestone #3: Implement contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by jodrack07**
For this milestone, I learned how  :
- Create forms with HTML5 validations. ✔️
-  Use the formspree to test the form ✔️
-  Understand the importance of UX. ✔️
-  Apply coding best practices ✔️
- Understand how to parse a Figma design to create a UI. ✔️

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -304,40 +304,42 @@ <h2>Contact me</h2>
...
-            ></textarea>
-          </div>
-            <button class="contact-btn" type="submit">Get In Touch</button>
+            <div>
+              <input
+                type="text"
+                name="name"
+                class="form-control"
+                id="name"
+                size="30"
```
Comment (sja-thedude) [inline index.html]: - [ ] Kindly change the `size` for input text to `max length` as per requirements.

File: style.css [hunk 2]
```diff
@@ -286,3 +286,87 @@ li {
...
   margin-right: 42px;
 }
+
+/* Contact-Form */
+
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-image: url('./images/contact-form-shape.png');
```
Comment (sja-thedude) [inline style.css:295]: - [ ] Kindly fix the background image for the entire section of the contact form as per the Figma design.
- Your design:
Screenshot 2022-03-23 at 12 19 47 AM

- Figma design:
Screenshot 2022-03-23 at 12 20 32 AM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @joseph07-drack, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good PR title and description
✔️ Good commit messages

## Required 
- [ ] Kindly add collaborator information to the readme file.
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Amrhub — APPROVED** [review]
# Approved 💯 ✅ 🥳 🎈 🎉 

Hi team, 👋🏻 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights ✅ 
Highly organized and clean code ✔️ 👨🏻‍💻 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by joseph07-drack** [comment]
Thank you so much @[Amrhub](https://github.com/Amrhub)

**Comment by jodrack07** [comment]
Thank you very much @Amrhub

**Pull request merged**